modules/reference/pages/cloud-shared-responsibility.adoc

```diff
@@ -0,0 +1,14 @@
+= Redpanda Cloud Shared Responsibility Model
+:description: Redpanda Cloud Shared Responsibility Model
+
+Redpanda Cloud shared responsibility model clarify the expectations around cloud resource ownership between Redpanda and Customer with respect to infrastructure, security and Redpanda service.
+
+Since BYOC is a fully managed offering running in customers' cloud accounts, a small set of responsibilities are shared. However, we strive to ensure the user experience is and remains a smooth SaaS experience.
```

Comment:
```suggestion
Since BYOC is a fully-managed offering running in customer cloud accounts, a small set of responsibilities are shared. However, Redpanda strives to ensure that the user experience is and remains a smooth SaaS experience.
```